src/drop-down/drop-down.component.ts

```diff
@@ -0,0 +1,375 @@
+import { CommonModule } from "@angular/common";
+import {
+    ChangeDetectorRef,
+    Component,
+    ContentChildren,
+    ElementRef,
+    EventEmitter,
+    forwardRef,
+    Input,
+    NgModule,
+    OnInit,
+    Output,
+    QueryList,
+    ViewChild
+} from "@angular/core";
+import { IToggleView } from "../core/navigation";
+import { IgxSelectionAPIService } from "../core/selection";
+import { IgxToggleDirective, IgxToggleModule } from "../directives/toggle/toggle.directive";
+import { IgxDropDownItemComponent } from "./drop-down-item.component";
+
+export interface ISelectionEventArgs {
+    oldSelection: IgxDropDownItemComponent;
+    newSelection: IgxDropDownItemComponent;
+}
+
+/**
+ * **Ignite UI for Angular DropDown** -
+ * [Documentation](https://www.infragistics.com/products/ignite-ui-angular/angular/components/drop-down.html)
+ *
+ * TODO: ADD DESCRIPTION HERE, SAME AS IN THE DOCS
```

Comment:
we have docs on PR, so we can update this section.

Reply:
Done